build.sh

```diff
@@ -321,6 +321,10 @@ if [[ `uname -a` =~ ^Darwin.* ]] && [[ `uname -a` =~ .*arm64$ ]]
 then
     CPU_ARCHITECTURE="ARM64"
     CMAKE_MORE_OPTIONS="${CMAKE_MORE_OPTIONS} -DBUILD_SSE2_CODEPATHS=OFF"
+elif [[ `uname -a` =~ .*riscv64.* ]]
+then
+    CPU_ARCHITECTURE="RISC-V 64"
+    CMAKE_MORE_OPTIONS="${CMAKE_MORE_OPTIONS} -DBUILD_SSE2_CODEPATHS=OFF"
```

Comment:
I actually prefer we removed this workaround and handle it in CMakeLists lists directly, see https://github.com/darktable-org/darktable/pull/12278

Reply:
No problem for me. FYI, `CMAKE_SYSTEM_PROCESSOR: riscv64` on RISC-V 64-bit platform.

Reply:
Rebased to the latest `master` branch and removed this change.